xija/tests/test_models.py

```diff
@@ -137,7 +137,7 @@ def test_minusz():
     mdl.make()
     mdl.calc()
 
-    regrfile = 'minusz.npz'
+    regrfile = abs_path('minusz.npz')
```

Comment:
Right, I just thought maybe the code to make it should be someplace else and/or should never try to write the file directly to arch.